### ISSUE TITLE
Table format support

### ISSUE THREAD

**Issue by georgelupu** [HadesArchitect/GrafanaCassandraDatasource#66]
Hi, in the current cassandra-grafana plugin implementation (as of Feb 2021) - only time-series data can be visualised in Grafana.

It would be great if the plugin would allow free form CQL in order to display for example reports in Table format like other SQL Grafana datasources allow (under "Format As", we should be able to choose Timeseries or Table). We should still have access to $__timeFrom and $__timeTo constructs in order to have a time filter in the free form query.

Is it possible to add this to the "to do" list?

Thank you!

**Comment by HadesArchitect** [comment]
Hi George,

thank you for the feedback, very appreciated! Sounds like a good idea. We will discuss and prioritize it this week.

**Comment by HadesArchitect** [comment]
FYI @futuarmo @Nortsx @IvanProdaiko94

**Comment by HadesArchitect** [comment]
https://grafana.com/docs/grafana/latest/panels/visualizations/table/

**Comment by HadesArchitect** [comment]
@Nortsx FYI

TL;DR Table view have to support any amounts of parameters

```
hey guys, i m working with the plugin and have a question... there is only the option to query the cassandra with 3 parameters in the select?
I m asking because  in my dashboard i want to see the min and max date for a especific metric
So for example this query:
    select min(timestamp_gmt_minus_3)
    from keyspace01.tcustomer_data_agg 
    where customer_id = $id_client and metadata_id = $id_metric
I get the error: "gocql: not enough columns to scan into: have 3 want 1"...
There is a workaround this?
```

**Comment by HadesArchitect** [comment]
@Nortsx Are you in discord? There is a discussion about it

**Comment by georgelupu** [comment]
@Nortsx - the plugin in its current implementation does not allow free form CQL, and you have to follow a strict formatting rule where the order of the columns matter in your select. 

For example this works fine:
SELECT
     string_column,
     double_column,
     timestamp_column
FROM
	table
WHERE
	timestamp_column > maxtimeuuid($__timeFrom) and timestamp_column < mintimeuuid($__timeTo) limit 5  allow filtering;

, but if you'll change the order of the first column with the second column, this grafana plugin will throw the following error: "can not unmarshal double into *string"

That is why i raised this issue #66 and i am asking if it is possible to implement a free form CQL that will be viewable in Grafana as table format and not time series (where a timestamp column must be provided in the Select at all times). 
For viewing Select data as Table examples, check other Grafana supported datasources like Postgres/MySQL where selected data can be viewed as Timeseries or as a Table. I hope this helps.

**Comment by Nortsx** [comment]
@georgelupu I got it! Will tackle this issue. @HadesArchitect can we postpone this feature a little bit?

**Comment by HadesArchitect** [comment]
In progress.

**Comment by HadesArchitect** [comment]
FE part is done, hope to implement BE part later on this week.

![image](https://user-images.githubusercontent.com/1742301/118414106-0ed9a080-b6a3-11eb-9144-6f305b8f5966.png)

**Comment by HadesArchitect** [comment]
Will be fixed by #124 when merged

**Comment by futuarmo** [comment]
Will be released in v2 in one or two days
